run_benchmarks.rb

```diff
@@ -300,29 +286,38 @@ def run_benchmarks(ruby_opts, name_filters, out_path)
 puts()
 
 # Table for the data we've gathered
-table = [["bench", "interp (ms)", "stddev (%)", "yjit (ms)", "stddev (%)", "interp/yjit", "1st itr"]]
+table  = [["bench", "interp (ms)", "stddev (%)", "yjit (ms)", "stddev (%)", "mjit (ms)", "stddev (%)", "interp/yjit", "yjit 1st itr", "mjit/yjit"]]
+format = [["%s",    "%.1f",        "%.1f",       "%.1f",      "%.1f",       "%.1f",      "%.1f",       "%.2f",        "%.2f",         "%.2f"]]
```

Comment:
Nice. You checked that the output is correct? Asking because I'm not sure how this will work with the table header row?